### PR TITLE
implement clean function

### LINKED ISSUE #9: Add clean() method
`handyman` could benefit from the addition of a `clean()` method which would be a wrapper around the [`del`](https://www.npmjs.com/package/del) Node module.

### PR DESCRIPTION
**Pull request by obuckley-kenzan**
resolve #9 , duplicates #10

### CODE REVIEW

**Comment by obuckley-kenzan** [comment]
cc: @jpoo90 / @paul-barry-kenzan 
If you like, please merge, and I will close the related issues / PRs

File: test/index.spec.js [hunk 1]
```diff
@@ -9,6 +10,20 @@ var util = require('gulp-util');
 
 describe('gulp-handyman', function () {
 
+  describe('clean', function () {
+    it('should delete the path passed as a param', function () {
+      // create temp dir for testing
```
Comment (paul-barry-kenzan) [inline test/index.spec.js]: Unneeded comments in this block

**Comment by obuckley-kenzan** [comment]
@paul-barry-kenzan 
updated

**Comment by paul-barry-kenzan** [comment]
@obuckley-kenzan 
This looks good but I don't have the power to merge

**Pull request merged**